platform/ios/scripts/package.sh

```diff
@@ -98,6 +99,17 @@ fi
 
 LIBS=(Mapbox.a)
 
+function copy_map_files {
```

Comment:
Having this as a named function is great if it’s called more than once, but I think doing this once would be possible if it were moved down to the `Copying library resources` section (where the bcsymbolmaps and such are copied).